### PR TITLE
Cancel JuMP-dev 2020 :(

### PR DESCRIPTION
**Pull request by odow**
Tested locally.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -149,4 +149,4 @@ sharing_links: # Appear at the bottom of single blog posts, add as desired. The
 
 # Site header notification content
 notification: |
-  [JuMP-dev 2020 registration delayed due to COVID-19](/meetings/louvain2020).
+  [JuMP-dev 2020 cancelled due to ongoing travel restrictions.](/jump-dev/2020/04/14/jump_dev_2020_cancellation/)
```
Comment (mlubin) [inline _config.yml:152]: due to COVID-19, not just the travel restrictions. Travel restrictions were a sufficient reason to cancel, but there are also many others like risks to participants' health. Same comment for other files.

File: _posts/2020-04-14-jump_dev_2020_cancellation.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Cancelled: JuMP-dev 2020"
+date:   2020-04-14
```
Comment (mlubin) [inline _posts/2020-04-14-jump_dev_2020_cancellation.md:4]: 2020-04-15

File: _posts/2020-04-14-jump_dev_2020_cancellation.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+categories: [jump-dev]
+---
+
+We regret to annouce that we have decided to cancel [JuMP-dev 2020](/meetings/louvain2020)
```
Comment (mlubin) [inline _posts/2020-04-14-jump_dev_2020_cancellation.md:8]: Maybe start by saying that we hope everyone is safe?

File: _posts/2020-04-14-jump_dev_2020_cancellation.md (hunk 2 above)
Comment (odow) [reply _posts/2020-04-14-jump_dev_2020_cancellation.md]: My brain is fried after the end of this day...

File: _posts/2020-04-15-jump_dev_2020_cancellation.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "Cancelled: JuMP-dev 2020"
+date:   2020-04-15
+categories: [jump-dev]
+---
+
+First, we hope that everyone is safe in these uncertain times. Our first
+priority will always be the health and well-being of the JuMP community.
+Therefore, we regret to annouce that we have decided to cancel [JuMP-dev 2020](/meetings/louvain2020).
```
Comment (mlubin) [inline _posts/2020-04-15-jump_dev_2020_cancellation.md:10]: Typo: announce?

Comment (odow) [reply _posts/2020-04-15-jump_dev_2020_cancellation.md]: I really am having a bad day.

**Pull request merged**